{"time":"2025-05-07T10:10:14.191620936+08:00","level":"INFO","msg":"stream: starting","core version":"0.19.9","symlink path":"/workspace/hide_lyf/DPOjudge/wandb/run-20250507_101014-z03c41ry/logs/debug-core.log"}
{"time":"2025-05-07T10:10:14.656895084+08:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp 35.186.228.49:443: connect: connection refused"}
{"time":"2025-05-07T10:10:16.996911195+08:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": dial tcp 35.186.228.49:443: connect: connection refused"}
